src/here-xyz.ts

```diff
@@ -1404,53 +1267,11 @@ program
     });
 
 function collate(result: Array<any>) {
-    return result.reduce((features: any, feature: any) => {
-        if (feature.type === "Feature") {
-            features.push(feature);
-        } else if (feature.type === "FeatureCollection") {
-            features = features.concat(feature.features);
-        } else {
-            console.log("Unknown type" + feature.type);
-        }
-        return features
-    }, []);
+    return xyzComm.collate(result);
```

Comment:
do we require this function?

Reply:
Have remove the reference of collate function from here